eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,13 @@ $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/r
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=cpp%2F&delimiter=%2F"
 
 # Parse out package publishing information given a nupkg ZIP format.
```

Comment:
This comment probably needs to be updated since vcpkg isn't a nupkg zip format but outside the scope of this PR.